examples/serial_dma_async.rs

```diff
@@ -56,8 +56,8 @@ fn main() -> ! {
         .split();
 
     // we only have two elements for each queue, so U2 is fine (size is max 2)
-    let mut rx_buffers: Queue<Pin<DmaBuffer>, 2> = Queue::new();
-    let mut tx_buffers: Queue<Pin<DmaBuffer>, 2> = Queue::new();
+    let mut rx_buffers: Queue<Pin<DmaBuffer>, 3> = Queue::new();
+    let mut tx_buffers: Queue<Pin<DmaBuffer>, 3> = Queue::new();
```

Comment:
Why does this need to be changed? Only 2 buffers are enqueued below, right?

(If it needs to be changed, then the comment above needs to be updated as well.)

Reply:
Thanks. Addressed in 11ddd02